src/pages/settings/Preferences/PreferencesPage.js

```diff
@@ -0,0 +1,137 @@
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import CONST from '../../../CONST';
+import * as User from '../../../libs/actions/User';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Switch from '../../../components/Switch';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import compose from '../../../libs/compose';
+import withEnvironment, {environmentPropTypes} from '../../../components/withEnvironment';
+import TestToolMenu from '../../../components/TestToolMenu';
+import MenuItemWithTopDescription from '../../../components/MenuItemWithTopDescription';
+
+const propTypes = {
+    /** The chat priority mode */
+    priorityMode: PropTypes.string,
+
+    /** The details about the user that is signed in */
+    user: PropTypes.shape({
+        /** Whether or not the user is subscribed to news updates */
+        isSubscribedToNewsletter: PropTypes.bool,
+        shouldUseStagingServer: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+    ...environmentPropTypes,
+};
+
+const defaultProps = {
+    priorityMode: CONST.PRIORITY_MODE.DEFAULT,
+    user: {},
+};
+
+const PreferencesPage = (props) => {
+    const priorityModes = {
+        default: {
+            value: CONST.PRIORITY_MODE.DEFAULT,
+            label: props.translate('priorityModesPage.mostRecent'),
+        },
+        gsd: {
+            value: CONST.PRIORITY_MODE.GSD,
+            label: props.translate('priorityModesPage.focus'),
+        },
+    };
+    const localesToLanguages = {
+        en: {
+            value: 'en',
+            label: props.translate('languagesPage.languages.english'),
+        },
+        es: {
+            value: 'es',
+            label: props.translate('languagesPage.languages.spanish'),
+        },
+    };
```

Comment:
I wonder if we should move these to CONST?

Reply:
fixed.